TCMSApp/src/client/app/tests/tests-create.controller.js

```diff
@@ -86,6 +86,11 @@
 
         }
 
+        function onCrtlEnterPress(event) {
+            if (event.charCode === 10) {
+                addStep();
```

Comment:
and also we need to move the cursor to the beginning of the next step here on ctrl+enter.